### PR TITLE
Support metadata on devnet and alphanet - Closes #1295

### LINKED ISSUE #1295: Support metadata on  devnet and alphanet
### Description

For seamless development, e2e integration and testing of the Lisk Wallets, Lisk Service needs to extend the metadata endpoints to support fetching the data and presenting results for the following networks as well: `devnet` and `alphanet`.

### Acceptance Criteria

- The metadata endpoints support indexing and querying metadata information for both `devnet` and `alphanet`
- Swagger specs are up-to-date
- Tests are up-to-date

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?
This PR resolves #1295 

### How was it solved?
- [x] Fix promise resolution in `downloadAndExtractTarball`
- [x] Extend metadata endpoints to support the following networks:
  - [x] devnet
  - [x] alphanet
- [x] Make `buildEventPayload` to be dynamic based on the supportedNetworks config
- [x] Update swagger
### How was it tested?
Locally against the changes in branch `1295-support-metadata-on-devnet-and-alphanet`

### CODE REVIEW

File: services/blockchain-app-registry/shared/utils/downloadUtils.js [hunk 1]
```diff
@@ -28,16 +28,17 @@ const logger = Logger();
...
+	logger.info(`Downloading and extracting file from ${url} to ${directoryPath}.`);
 	getHTTPProtocolByURL(url).get(url, (response) => {
 		if (response.statusCode === 200) {
 			response.pipe(tar.extract({ cwd: directoryPath }));
 			response.on('error', async (err) => reject(new Error(err)));
 			response.on('end', async () => {
-				logger.info('File downloaded successfully');
-				resolve();
+				logger.info('File downloaded and extracted successfully.');
+				setTimeout(resolve, 1); // Since the promise resolves earlier than expected
```
Comment (priojeetpriyom) [inline services/blockchain-app-registry/shared/utils/downloadUtils.js:38]: Could it be prone to race condition?

Comment (sameersubudhi) [reply services/blockchain-app-registry/shared/utils/downloadUtils.js:38]: JS is single threaded, I'm not sure if this comment is valid here? Could you please elaborate your statement?

Comment (priojeetpriyom) [reply services/blockchain-app-registry/shared/utils/downloadUtils.js:38]: I mean we are delaying execution by 1ms, what if the server is under pressure and it is not enough?

Comment (sameersubudhi) [reply services/blockchain-app-registry/shared/utils/downloadUtils.js]: Increase the delay to 100ms. This should be ample.

**Review by priojeetpriyom — APPROVED** [review]

File: services/blockchain-app-registry/shared/utils/downloadRepository.js [hunk 2]
```diff
@@ -156,34 +156,25 @@ const getUniqueNetworkAppDirPairs = async (files) => {
...
-		{ concurrency: betanetFilesUpdated.length },
-	);
+const buildEventPayload = async (allFilesModified) => {
+	const eventPayload = {};
+
+	const { supportedNetworks } = config;
+	const numSupportedNetworks = supportedNetworks.length;
+	for (let index = 0; index < numSupportedNetworks; index++) {
+		/* eslint-disable no-await-in-loop */
+		const networkType = supportedNetworks[index];
```
Comment (priojeetpriyom) [inline services/blockchain-app-registry/shared/utils/downloadRepository.js:166]: could be written with forEach as well.

**Pull request merged**